CheckIn/AttendanceReports/AttendanceReportDebug.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Attendance Report</title>
+    <style></style>
+</head>
+
+<body>
+    <ul>
+        {%- for row in Rows -%}
+        {%- assign rowJson = row | ToJSON -%}
+        {%- assign realRow = rowJson | FromJSON -%}
```

Comment:
@treyhendon Is this a trick to merge documents I haven't known about? Does it not parse the objects properly initially that you have to go to JSON and then back from JSON to get the real object?

Reply:
I'm not sure if it's a trick or hack. :)  I was not able to properly parse and loop through the "other attendance list" property in each row until I did this.  It could be a v11 issue - not totally sure.  The biggest issue I couldn't get any objects to properly map back to entities which is why there are so many assigns throughout the lava.